### DISCUSSION TITLE
Some advance official documentation pages (e.g. https://fastapi.tiangolo.com/advanced/custom-response/) are not working by 05.06.24

### DISCUSSION THREAD

**Discussion by AlexanderMishutkin** [fastapi/fastapi#11685] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
curl https://fastapi.tiangolo.com/advanced/custom-response/
```

### Description

While reading the documentation I found out that some pages are simply not working. In example:
- https://fastapi.tiangolo.com/advanced/using-request-directly/
- https://fastapi.tiangolo.com/advanced/custom-response/

### Operating System

Other

### Operating System Details

Browser: Chrom

### FastAPI Version

latest

### Pydantic Version

latest

### Python Version

latest

**Comment by Gnosnay** [comment]
same to [https://[fastapi.tiangolo.com](/)/tutorial/dependencies/dependencies-in-path-operation-decorators/](https://fastapi.tiangolo.com/tutorial/dependencies/dependencies-in-path-operation-decorators)

**Comment by jpenna** [comment]
There are many not working. Everything under and including `deployment` for instance: https://fastapi.tiangolo.com/deployment/

**Comment by AlexanderMishutkin** [comment]
Only temporary solution I can suggest for now is to use Web Archive https://web.archive.org/web/20240324095137/https://fastapi.tiangolo.com/tutorial/

**Comment by jereflit** [comment, marked as answer]
Back to normal, I suggest we can close this.
